src/App.js

```diff
@@ -19,7 +20,13 @@ export default function App() {
   return (
     <Router>
       <Routes>
-        <Route path="/" element={<h2>Just localhost</h2>} />
+        <Route
+          path="/"
+          element={
+            <Onboarding1 />
+            // <h2>Just localhost</h2>
+          }
+        />
         <Route path="/sometest" element={<Testing />} />
         <Route path="/rsvp-preview" element={<RSVP />} />
         <Route path="/invite" element={<InvitationForm />} />
```

Comment:
Would there be a good use case for nested routing here and in other parts of the App, as I imagine there would be a common element (like Navbar) that should remain at the top of the page? Idea of nested routing is that going to a child route does not require a full reload of the page. I recognized this was already implemented in a prior merge, but just a thought as I look at the basic routing.

https://reactrouter.com/docs/en/v6/getting-started/tutorial#nested-routes